python/lsst/ts/m2/csc.py

```diff
@@ -525,17 +545,25 @@ async def do_enable(self, data: salobj.BaseMsgType) -> None:
                     retry_times=self.ilc_retry_times,  # type: ignore[arg-type]
                 )
 
-            except RuntimeError:
-                # TODO: Publish the list of failed ILCs after the DM-40146 is
-                # done.
+            except RuntimeError as error:
                 await self._basic_cleanup_and_power_off_motor()
+
+                error.add_note("Power-off the motor.")
                 raise
 
-        await self._execute_command(
-            self.controller_cell.set_closed_loop_control_mode,
-            MTM2.ClosedLoopControlMode.OpenLoop,
-            timeout=self.COMMAND_TIMEOUT_LONG,
-        )
+        try:
+            await self._execute_command(
+                self.controller_cell.set_closed_loop_control_mode,
+                MTM2.ClosedLoopControlMode.OpenLoop,
+                timeout=self.COMMAND_TIMEOUT_LONG,
+            )
+
+        except RuntimeError as error:
+            if self._exists_error_in_controller():
+                error.add_note(
+                    "The existed error blocks the transition to open-loop control."
```

Comment:
Consider rephrasing:

```
Cannot transition to open-loop control due to existing errors.
```

Also, is there any hint on what people can/should do?

Reply:
Updated in the commit: [cbe9944](https://github.com/lsst-ts/ts_m2/pull/56/commits/cbe99449d8ee63e2b8c583e451cee4ffd355c7ff). Thanks!